### ISSUE TITLE
[1.20.1] Abydos City generate empty?

### ISSUE THREAD

**Issue by drakray** [Povstalec/StargateJourney#38]
I found walls without anything inside, and using Minutor shows that it should be a sgjourney:city/abydos_city, but there is just a wall
![2023-11-07 11_15_56-Minutor - Open Roads](https://github.com/Povstalec/StargateJourney/assets/4303126/6f0209bb-c9be-453c-ad4b-82daf8819b14)

EDIT: also, overlap

**Comment by drakray** [comment]
Unrelated, but I also found that the dimensions definition goes from -64 to 384, but the bedrock is at y0
some things like Geode might generate partially under the bedrock(though that may be my datapack, unsure if they naturally generate there)

**Comment by Povstalec** [comment]
Yeah, the wall is there because I haven't added any buildings inside the city yet.

I'm not sure why the dimension definition is that big, but thanks for informing me.

**Comment by Radon8472** [comment]
> Yeah, the wall is there because I haven't added any buildings inside the city yet.
> 
> I'm not sure why the dimension definition is that big, but thanks for informing me.

------------------------------------------
I found simple villages generating on Abydos.
Maybe you could generate a simple village inside the City in meantime and replace this when you have added your real city buildings

**Comment by Povstalec** [comment]
Having a Village generate there without taking care of how exactly it generates would most definitely cause parts of the structure to intersect and create weird amalgamations. And going in figuring out how exactly it should generate in order for that to not happen would probably take the about as much time as creating a custom city inside anyway.
